packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -186,13 +187,23 @@ export async function internalGetErc1155AllTokens(
 }
 
 export async function internalGetErc1155ByTokenId(
+  readonlyDBConn: Pool,
+  cde_id: number,
+  token_id: bigint
+): Promise<ICdeErc1155GetByTokenIdResult | null> {
+  return (
+    await cdeErc1155GetByTokenId.run({ cde_id, token_id: String(token_id) }, readonlyDBConn)
+  )[0];
+}
+
+export async function internalGetErc1155ByTokenIdAndWallet(
   readonlyDBConn: Pool,
   cde_id: number,
   wallet_address: string,
   token_id: bigint
 ): Promise<ICdeErc1155GetByTokenIdResult | null> {
```

Comment:
The types are structurally the same, but maybe for max correctness this (and `getErc1155ByTokenIdAndWallet` in the other file) should be returning `ICdeErc1155GetByTokenIdAndWalletResult`.